client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -99,16 +98,23 @@ export const StockLineForm: FC<StockLineFormProps> = ({
         flexDirection="column"
         gap={1}
       >
-        <TextWithLabelRow
+        <StyledInputRow
           label={t('label.num-packs')}
-          text={String(draft.totalNumberOfPacks)}
-          textProps={{ textAlign: 'end' }}
+          Input={
+            <NumericTextInput
+              autoFocus
+              disabled={!packEditable}
```

Comment:
Disabling these inputs when not editable... changes the stock line edit UI to look like this:

![Screenshot 2024-04-10 at 3 27 59 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/8bbf716f-b53b-4cff-868a-e3797124fa61)

Instead of before it was like this:

![Screenshot 2024-04-10 at 3 28 09 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/445315ab-4b18-43de-94e5-05de246f2854)

I don't think it's a biggie... though also aware of making any changes that should be hidden in 2.0.0 👀